backend/packages/Upgrade/package.json

```diff
@@ -115,5 +114,9 @@
         "uuid": "^3.3.3",
         "winston": "^3.2.1",
         "winston-splunk-httplogger": "^3.1.0"
+    },
+    "overrides": {
+        "class-validator": "^0.14.0",
+        "typeorm": "^0.2.20"
     }
```

Comment:
Can we use the references to our specified library versions, so that the overrides don't get out of sync when we upgrade them?

```
"overrides": {
        "class-validator": "$class-validator",
        "typeorm": "$typeorm"
}
```